[RDY]vim-patch:8.1.{1614,1632,1634,1639}

janlazo opened the pull request.

janlazo committed:
vim-patch:8.1.1614: 'numberwidth' can only go up to 10

Problem:    'numberwidth' can only go up to 10.
Solution:   Allow up to 20. (Charlie Stanton, closes vim/vim#4584)
https://github.com/vim/vim/commit/f8a071265535b8cc43e50a81f4d5049883ca50e4

blueyed commented:
Legit failure:
```
[  FAILED  ] 1 test, listed below:
[  FAILED  ] ...ects/neovim/test/functional\options\num_options_spec.lua @ 44: :set validation setlocal and setglobal validate values
...ects/neovim/test/functional\options\num_options_spec.lua:9: Expected objects to be the same.
Passed in:
(nil)
Expected:
(string) 'E474'
stack traceback:
	...ects/neovim/test/functional\options\num_options_spec.lua:9: in function 'should_fail'
	...ects/neovim/test/functional\options\num_options_spec.lua:77: in function <...ects/neovim/test/functional\options\num_options_spec.lua:44>
```

janlazo committed:
vim-patch:8.1.1632: build with EXITFREE but without +arabic fails

Problem:    Build with EXITFREE but without +arabic fails.
Solution:   Rename the function and adjust #ifdefs. (closes vim/vim#4613)
https://github.com/vim/vim/commit/48ac671fe5cb5a7c2d5263d2f122e5e903022e30

janlazo committed:
vim-patch:8.1.1634: terminal test fails when term_getansicolors() is missing

Problem:    Terminal test fails when term_getansicolors() is missing.
            Diff test fails without +rightleft.  (Dominique Pelle)
Solution:   Check if term_getansicolors() is supported. (closes vim/vim#4597)
https://github.com/vim/vim/commit/981d9dc8f3d7a470eb3be7ba60fe1c1f31e7f470

janlazo committed:
vim-patch:8.1.1639: changing an autoload name into a file name is inefficient

Problem:    Changing an autoload name into a script file name is inefficient.
Solution:   Remember the last replaced #. (Ozaki Kiichi, closes vim/vim#4618)
https://github.com/vim/vim/commit/28fc247f8d94a1cfabbcf2691ca942bde96f6d2f
